src/components/Header.tsx

```diff
@@ -106,7 +109,13 @@ const HeaderLinks = () => {
 const Header = (props: BoxProps) => {
   const isClientSide = useIsClientSide()
   return (
-    <Box as="header" bg="gray.700" color="white" zIndex="banner" {...props}>
+    <Box
+      as="header"
+      zIndex="banner"
+      {...props}
+      borderTop={{ base: '4px', sm: '8px' }}
+      borderColor={{ base: 'fuchsia.700', sm: 'fuchsia.700' }}
```

Comment:
I needed to change `borderColor` as well, otherwise the `borderTop` was using a black color instead of the fuchsia shade we want...

Unsure if bug in Chakra UI, or if this is just the way it needs to be when changing a property to be responsive....

Reply:
If you just set `borderColor="fuchsia.700"` then you get a black border? That's weird.

Reply:
I've pushed a commit that shows the bad behavior. 😄

Reply:
I've checked the issue in the preview platform. I've seen Chakra is generating another CSS class for desktop, but I've found the issue is on our side 😅 

The `borderTop` prop is a shortcut for setting border style, color, and width at once (same as `border-top` CSS property actually). Since we are passing just the width, Chakra is filling the other values with the default border style and color.

The proper way to set this then is using `borderTopWidth` instead:

```js
      borderTopWidth={{ base: '4px', sm: '8px' }}
      borderColor="fuchsia.700"
```

You can find all Chakra Border props [here](https://chakra-ui.com/docs/features/style-props#borders).

Reply:
Ah we were basically taking a "shortcut" instead of using the proper separate variables. 😄